Cargo.toml

```diff
@@ -24,11 +24,12 @@ appveyor = { repository = "chronotope/chrono" }
 name = "chrono"
 
 [features]
-default = ["clock", "std"]
+default = ["clock", "std", "locales"]
```

Comment:
I don't know that this should be a default feature, but I _do_ think that it should be called out in the README as a major feature, there isn't a clear section for features right now, I'll add one and make sure that this is prominent.

Reply:
done